src/dapp/README.md

```diff
@@ -189,6 +192,26 @@ which demonstrates that if we give the password `42`, it is possible to withdraw
 
 For more reading on property based testing and symbolic execution, see [this tutorial on the Ethereum Foundation blog](https://fv.ethereum.org/2020/12/11/symbolic-execution-with-ds-test/).
 
+### Invariant testing
+
+While other forms of tests are always run against the post state of the `setUp()` function in the testing contract,
+it can be also be useful to check whether a property is satisfied at any possible contract state. This can be done with
+the `invariant*` testing type. When running an invariant test, hevm will invoke any state mutating function from any 
+non-testing contract available after the `setUp()` function has been run, checking the `invariant*` after each run.
+
+The `--depth` paramenter determines how many transactions deep each test will run, while the `--fuzz-runs` parameter 
+determines how many times the whole process is repeated.
```

Comment:
```suggestion
determines how many times the whole process is repeated.

Note that unlike the other test types, `invariant*` tests do not treat reverting calls as a failure. The goal of invariant tests is to find a state change that results in a violation of the assertions defined in the body of the test method, and since reverts do not result in a state change, they can be safely ignored.
```

Reply:
well, reverts in the testing function are still treated as failure. Just not any of the exploration calls

Reply:
ah true, what about:

```suggestion
determines how many times the whole process is repeated.

Note that a revert in any of the randomly generated call will not trigger a test failure. The goal of invariant tests is to find a state change that results in a violation of the assertions defined in the body of the test method, and since reverts do not result in a state change, they can be safely ignored. Reverts within the body of the `invariant*` test method will however still cause a test failure.
```

Reply:
nice, thx. Added